bin/oneclick_library_delta

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+"""Update the circulation manager server with new books from the
+open access content server."""
+import os
+import sys
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+from core.scripts import OneClickDeltaScript
+
+from api.config import Configuration
+Configuration.load()
+library_id = Configuration.integration(Configuration.ONECLICK_INTEGRATION).get("library_id", None)
+
+print "Applying catalog delta of library #%s" % library_id
+OneClickDeltaScript().run()
```

Comment:
If you need to print this string, put it inside `OneClickDeltaScript` and have it use `self.log`.